### ISSUE TITLE
[Bug] MangaHere : 404 error for select titles

### ISSUE THREAD

**Issue by QuietusWolfe** [inorichi/tachiyomi-extensions#2292]
* Tachiyomi extension version: 1.2.12
* Tachiyomi version: 0.8.4

When trying to open certain titles, while the app will find the chapters in question, any attempt to open them results in a 404 error. 

Error seems linked to MangaHere's use of a "click here to reveal' link to hid 'adult' or mature themed manga and opening in webview and selecting that link doesn't allow app to bypass it.

(if this is a dupe issue, then my bad. I did search and didn't find it)

**Comment by happywillow0** [comment]
The issue comes actually from the slight difference between the chapter url of the desktop and mobile versions of the website for mangas that have super long names (I think)

The desktop version has encryption / obfuscation which hides the images from being pulled. 
I found that a easy workaround was to just use the mobile website. At the time, I took a minimal change in the code which takes the desktop site and changes it to the mobile. (changes www.mangahere to m.mangahere)

Two options to take would be to put keep the desktop site and put back the de-obfusaction code, or to go though and covert everything else in the extension to the mobile site.
